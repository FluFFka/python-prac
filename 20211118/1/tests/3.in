@objcount
class A:
    def __init__(self, q, qq=1):
        self.q = q
        self.qq = qq
    def __del__(self):
        print(self.counter)
        print(self.q)
        print(self.qq)

a = A('123')
b = A(123, 444)
c = A(None, qq='YEAH')
print(A.counter)
del a
del b
print(A.counter)
print(c.q)
c.q = 13333
print(c.q)
del c
print(A.counter)
print(A.counter)
