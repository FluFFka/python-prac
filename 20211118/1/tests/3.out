3
2
123
1
1
123
444
1
None
13333
0
13333
YEAH
0
0
